.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  issues:
+    types: [ opened, reopened ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Python stuff
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Checkout mbed-os-scripts repo
```

Comment:
as t his is yaml, we can add a comment `the repository is private`. 

A question will be if we can move the script to a public repo. I would say this one should be part of this repository, we can take that separately.